src/photosui.cs

```diff
@@ -580,5 +592,17 @@ interface PHPhotoLibrary_PhotosUISupport
 	{
 		[Export ("presentLimitedLibraryPickerFromViewController:")]
 		void PresentLimitedLibraryPicker (UIViewController controller);
+
+		[NoWatch, NoTV, NoMac, iOS (15,0), MacCatalyst (15,0)]
+		[Export ("presentLimitedLibraryPickerFromViewController:completionHandler:")]
+		void PresentLimitedLibraryPicker (UIViewController controller, Action<NSArray<NSString>> completionHandler);
```

Comment:
The API clash with `string[] PreselectedAssetIdentifiers { get; set; }`

In one case you force the use of `NSArray<NSString>` as the parameter of the `completionHandler`.
But you expose the pre-selection assets as a `string[]`. 
That can force (another) conversion of the data inside user code.

Using `Action<string[]>` should work fine.

Reply:
@spouliot  Can you explain this a little more?
I think I get that this action is taking an NSArray<NSString> and PreselectedAssetIdentifiers returns a string[]
but how are these two related? Is this a general thing within files to not sure both string[] and NSArray<NSString> and just one of them?

Reply:
> but how are these two related? I

I was suspicious reading the diff... so I looked at the docs/headers and they seems related.

Also we generally do not use `NSArray<NSString>` inside bindings (grep src) but prefer `string[]`

`NSString` is more useful (even required in some cases) when dealing with constants (since they can be compared by pointers, inside native code, and not by value).